config/initializers/ahoy.rb

```diff
@@ -6,7 +6,7 @@
 module Ahoy
   class Store < Ahoy::Stores::LogStore
     def exclude?
-      return if FeatureManagement.enable_load_testing_mode?
+      return true if FeatureManagement.enable_load_testing_mode?
```

Comment:
If by "analytics seem to be broken", you mean that they aren't being written to the `events.log` file on the server, I'm pretty sure this is not the cause. Analytics are being written correctly in development. All this method does is determine whether or not certain visitors should be excluded. By default, [Ahoy excludes what it thinks are bots](https://github.com/ankane/ahoy/blob/52c3b6a768c98ef10078f75dec6a11799e8473b8/lib/ahoy/stores/base_store.rb#L37-L39). When we run the load testing scripts, Ahoy treats the Locust user agent as a bot, and therefore doesn't write analytics events. I wanted to allow events to be written during load testing to confirm that the proper sequence of events is being followed.

The `return` here says that if load testing mode is turned on, don't bother going through any logic to determine whether or not the request should be recorded. In other words, it tells Ahoy to go ahead and record all requests.

Yes, semantically, since this is a predicate method, it should return a boolean, and if we want to make sure a boolean was returned, what we want here is to return `false` because we don't want to exclude anything during load testing.

If the issue is that Ahoy somehow thinks that all requests to the production server are coming from bots, one way to test that is to set `enable_load_testing_mode` to `'true'` on the production server and then see if events are being recorded.